docs/sections/footer.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no">
+  <link rel="stylesheet" href="/cocoandeve-styleguides/css/main.critical.css">
+  <link rel="stylesheet" href="/cocoandeve-styleguides/css/main.css">
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js"></script>
+  <script src="/cocoandeve-styleguides/js/bootstrap.js"></script>
+  <script src="/cocoandeve-styleguides/js/main.js"></script>
+</head>
+
+<body class="">
+  <div class="container mt-4">
+    <h1>Footer</h1>
+  </div>
+  <footer class="pt-md-6 pb-md-6 pt-2 pt-lg-4 pb-1 mobile-wrapper">
+    <div class="container">
+      <div class="footer-grid">
+        <div class="footer-grid__newsletter">
+          <h5 class="h2">Newsletter</h5>
+          <p>Receive exclusive offers, promotions and beauty tips via email.</p>
+          <form>
+              <div class="input-group mb-3">
+                  <input type="text" class="form-control" placeholder="Enter your email" aria-label="Enter your email">
+                  <div class="input-group-append">
+                      <button class="btn btn-primary" type="button">Subscribe</button>
+                  </div>
+              </div>
+          </form>
+          <p>You can unsubscribe at any time. Please read our <a href="/pages/privacy-policy-new" title="Privacy Policy">Privacy Policy</a> for more information about how we use your data.</p>
+        </div>
+        <div class="footer-grid__about-us">
+          <h5 class="h2">About Us</h5>
+          <ul class="list-unstyled">
+            <li><a href="">Our Story</a></li>
+            <li><a href="">Stockist</a></li>
+            <li><a href="">Affiliates</a></li>
+            <li><a href="">Blog</a></li>
+          </ul>
+        </div>
+        <div class="footer-grid__help-links">
+          <h5 class="h2">Help</h5>
+          <ul class="list-unstyled">
+            <li><a href="">Track My Order</a></li>
+            <li><a href="">FAQ</a></li>
+            <li><a href="">Shipping & Refund</a></li>
+            <li><a href="">Contact Us</a></li>
+            <li><a href="">Terms & Conditions</a></li>
+            <li><a href="">Privacy Policy</a></li>
+          </ul>
+        </div>
+        <div class="footer-grid__support">
+          <h5 class="h2">Support</h5>
+          <p>Got a question? Contact us <a href=""><u>here</u></a></p>
+        </div>
+        <div class="footer-grid__follow-us">
+          <h2>Follow Us</h2>
+          <a href="https://www.instagram.com/cocoandeve/">
```

Comment:
A and I can be 1 unique element. Use only A. Also on hover they should now be underlined. Please also use target="_blank" for these kind of links